rir/src/utils/FunctionHandle.h

```diff
@@ -32,25 +32,25 @@ class FunctionHandle {
         SEXP store = Rf_allocVector(EXTERNALSXP, initialSize);
         void* payload = INTEGER(store);
 
-        TYPEOF(store) = EXTERNALSXP;
-        R_PreserveObject(store);
-
-        Function* function = new (payload) Function;
```

Comment:
Not sure why placement new here?

Reply:
I'd say it is more idiomatic to use placement new here. but of course since Function is a C struct, it should have no effect.

Reply:
Ok, reverted the change.